tests/devices/qutrit_mixed/test_qutrit_mixed_tracking.py

```diff
@@ -0,0 +1,265 @@
+
+    def test_tracking_batched_execution(self):
+        """Test the number of times the device is executed over a QNode's
+        lifetime is tracked by the device's tracker."""
+
+        dev_1 = qml.device("default.qutrit.mixed", wires=2)
+
+        def circuit_1(x, y):
+            qml.TRX(x, wires=[0])
+            qml.TRY(y, wires=[1])
+            qml.TAdd(wires=[0, 1])
+            return qml.expval(qml.GellMann(0, 3) @ qml.GellMann(1, 4))
+
+        node_1 = qml.QNode(circuit_1, dev_1)
+        num_evals_1 = 10
+
+        with qml.Tracker(dev_1, persistent=True) as tracker1:
+            for _ in range(num_evals_1):
+                node_1(0.432, np.array([0.12, 0.5, 3.2]))
+        assert tracker1.totals["executions"] == 3 * num_evals_1
+        assert tracker1.totals["simulations"] == num_evals_1
+
+        # test a second instance of a default qutrit mixed device
+        dev_2 = qml.device("default.qutrit.mixed", wires=2)
+
+        def circuit_2(x):
+            qml.TRX(x, wires=[0])
+            qml.TAdd(wires=[0, 1])
+            return qml.expval(qml.GellMann(0, 3) @ qml.GellMann(1, 4))
+
+        node_2 = qml.QNode(circuit_2, dev_2)
+        num_evals_2 = 5
+
+        with qml.Tracker(dev_2) as tracker2:
+            for _ in range(num_evals_2):
+                node_2(np.array([0.432, 0.61, 8.2]))
+        assert tracker2.totals["simulations"] == num_evals_2
+        assert tracker2.totals["executions"] == 3 * num_evals_2
+
+        # test a new circuit on an existing instance of a qutrit mixed device
+        def circuit_3(y):
+            qml.TRY(y, wires=[1])
+            qml.TAdd(wires=[0, 1])
+            return qml.expval(qml.GellMann(0, 3) @ qml.GellMann(1, 4))
+
+        node_3 = qml.QNode(circuit_3, dev_1)
+        num_evals_3 = 7
+
+        with tracker1:
+            for _ in range(num_evals_3):
+                node_3(np.array([0.12, 1.214]))
+        assert tracker1.totals["simulations"] == num_evals_1 + num_evals_3
+        assert tracker1.totals["executions"] == 3 * num_evals_1 + 2 * num_evals_3
+
+
+shot_testing_combos = [
+    # expval combinations
+    ([qml.expval(qml.GellMann(0, 1))], 1, 10),
+    ([qml.expval(qml.GellMann(0, 1)), qml.expval(qml.GellMann(0, 2))], 2, 20),
+    # Hamiltonian test cases
+    ([qml.expval(qml.Hamiltonian([1, 1], [qml.GellMann(0, 1), qml.GellMann(1, 5)]))], 2, 20),
+    # op arithmetic test cases
+    ([qml.expval(qml.sum(qml.GellMann(0, 1), qml.GellMann(1, 4)))], 2, 20),
+    (
+        [
+            qml.expval(qml.prod(qml.GellMann(0, 1), qml.GellMann(1, 4))),
+            qml.expval(qml.prod(qml.GellMann(1, 4), qml.GellMann(2, 7))),
+        ],
+        2,
+        20,
+    ),
+    # computational basis measurements
+    ([qml.sample(wires=(0, 1))], 1, 10),
+    ([qml.sample(wires=(0, 1)), qml.expval(qml.GellMann(0, 1))], 2, 20),
+]
+
+
+@pytest.mark.parametrize("mps, expected_exec, expected_shots", shot_testing_combos)
```

Comment:
Since these are all related tests, they can be included together in a new class.